src/components/Input/Input.stories.tsx

```diff
@@ -1,60 +1,152 @@
 import * as React from "react";
 
 import Input from "./Input";
+import { InputTypes } from "./InputTypes";
+import Label from "../Label/Label";
+import Button from "../Button/Button";
+import { ButtonTypes } from "../Button/ButtonTypes";
+import HelperErrorText from "../HelperErrorText/HelperErrorText";
+import { text, boolean, select } from "@storybook/addon-knobs";
+import { action } from "@storybook/addon-actions";
 
 export default {
     title: "Input",
     component: Input,
 };
 
+let showLabel, showHelperText;
+
 export const input = () => (
-    <Input
-        labelId="blah"
-        labelContent={<>Label</>}
-        inputId="blah"
-        isRequired={false}
-    ></Input>
+    <>
+        {boolean("Show Label", true) ? (showLabel = true) : (showLabel = false)}
+        {showLabel && (
+            <Label
+                htmlFor="inputID"
+                optReqFlag={select(
+                    "Optional/Required Flag",
+                    ["Required", "Optional", ""],
+                    "Required"
+                )}
+                id={"label"}
+            >
+                {text("Input Label", "Choose your islander name: ")}
+            </Label>
+        )}
+        <Input
+            id="inputID"
+            ariaLabel="Input Label"
+            helperTextId={"helperText"}
+            required={boolean("Input Required", false)}
+            placeholder={text("Input Placeholder", "CoolPerson99")}
+            type={select("Input Type", InputTypes, InputTypes.text)}
```

Comment:
Is this in preparation to get the `input` to be an element that supports all inputs?  Or is this still going to only support text or number still?  

Maybe I'm misunderstanding the purpose of knobs - are they not supposed to all be expressive?  Considering that we may be sharing the storybook outside of engineering, how can we communicate which knobs produce expressive and useful results, and which don't?

Reply:
Looking through the Metronome documentation, I'm not seeing any wording that the DS.Input should only accept text/numbers—do you know where you heard that requirement?

I'm not sure expressive and useful are the right words here—maybe "designed" and "not designed". All of the knobs here are still useful in that they can show UX what is and isn't designed, and they are all expressive in that they are all valid HTML and options that the HTML spec gives developers.

That said I think we should eliminate the other InputTypes all the same if it's going to cause confusion. I'll rip the extras out for now but I'd like to leave it as an enum so we can add to it over time.

Reply:
(Ended up also leaving in the `password` type... if the designers want anything other than the default I want them to be thinking about that before we get to any Header refactor.)

Reply:
> I'm not seeing any wording that the DS.Input should only accept text/numbers—do you know where you heard that requirement?

I didn't -- it was more me wondering, is `DS.Input` specific to stuff you type on your keyboard, or should it also encompass <select> and <radio> and <checkbox> ?  The changes to `InputTypes` makes this clear now.